src/parser/core/modules/Gauge/Gauge.tsx

```diff
@@ -48,16 +67,34 @@ export class Gauge extends Analyser {
 		return gauge
 	}
 
-	private onDeath() {
+	protected onDeath() {
 		this.pauseGeneration = true
 		this.gauges.forEach(gauge => gauge.reset())
 	}
 
-	private onRaise() {
+	protected onRaise() {
 		this.pauseGeneration = false
 		this.gauges.forEach(gauge => gauge.raise())
 	}
```

Comment:
I don't see any users being updated in this PR to use these - what's the reasoning behind exposing them?  If we are going to expose these, we should move the pauseGeneration infrastructure out to specifically the dancer gauges and make that behavior an implementation detail of gauge modules that need it.